akka-stream/src/main/scala/akka/stream/io/IOResult.scala

```diff
@@ -0,0 +1,33 @@
+package akka.stream.io
+
+import akka.Done
+import scala.util.{ Failure, Success, Try }
+
+/**
+ * Holds a result of an IO operation.
+ *
+ * @param count A number of IO operations performed.
```

Comment:
Bytes might not be available (for a future `Sink.count`) or the right abstraction (for text processing).

+1 for explicitly documenting the meaning to be context-dependent.